uniffi/src/interface/mod.rs

```diff
@@ -699,10 +742,50 @@ impl APIConverter<Method> for weedle::interface::OperationInterfaceMember<'_> {
                 weedle::types::ReturnType::Type(t) => Some(t.resolve_type_definition(ci)?),
             },
             ffi_func: Default::default(),
+            throws: match &self.attributes {
+                Some(attr) => attr.body.list.find_throws(ci)?,
```

Comment:
This will error out when if sees other attributes (Since we don't support them) I'll add that!